docs/DATA_INGRESS.md

```diff
@@ -0,0 +1,24 @@
+# SDS Data Ingress
+
+These samples showcase some of the commonly performed data ingress options into SDS. These include OMF and SDS calls.
+
+## [OMF Ingress](https://github.com/osisoft/sample-ocs-omf_ingress-dotnet)
+
+This example highlights the creation of an OMF ingress connection. It also shows sending OMF messages. Please see the [OMF Ingress](https://github.com/osisoft/sample-ocs-omf_ingress-dotnet) readme for more information on this sample.
```

Comment:
Wouldn't be a bad idea to change `example` to `sample` everywhere, too, if you wanted to.